dockstore-webservice/src/main/resources/migrations.1.16.0.xml

```diff
@@ -423,4 +423,13 @@
             </column>
         </addColumn>
     </changeSet>
+    <changeSet author="dyuen (generated)" id="move_aitopicprocessed_to_versionmetadata">
+        <addColumn tableName="version_metadata">
+            <column defaultValueBoolean="false" name="aitopicprocessed" type="bool"/>
+        </addColumn>
+        <dropColumn tableName="tag" columnName="aitopicprocessed">
+        </dropColumn>
+        <dropColumn tableName="workflowversion" columnName="aitopicprocessed">
+        </dropColumn>
+    </changeSet>
```

Comment:
This migration will reset all of the `aitopicprocessed` values to false.  Is that ok?  We could try to copy the existing values over, not sure it's worth it tho...

Reply:
Yes that's okay. We only have about 23 AI topics in QA which can be easily re-generated after the column is dropped